### PR TITLE
chore: add pricing url and customer portal url to env var

### PR DESCRIPTION
**Pull request by berzerkeer**
## Description

 - The customer portal URL and pricing page URL should come from env vars. So that release can point to release customer portal and prod can use prod customer portal URL.

Fixes [#487](https://github.com/appsmithorg/cloud-services/issues/487)

## Type of change

- Chore

## How Has This Been Tested?

- Manual

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/ce/configs/index.ts [hunk 1]
```diff
@@ -131,6 +133,8 @@ export const getConfigsFromEnvVars = (): INJECTED_CONFIGS => {
       .APPSMITH_DISABLE_IFRAME_WIDGET_SANDBOX
       ? process.env.APPSMITH_DISABLE_IFRAME_WIDGET_SANDBOX.length > 0
       : false,
+    pricingUrl: process.env.REACT_APP_PRICING_URL || "",
```
Comment (dipyamanbiswas07) [inline app/client/src/ce/configs/index.ts:136]: do we need "" as fallback? might result in some breakages

Comment (berzerkeer) [reply app/client/src/ce/configs/index.ts:136]: Here, we check for the pricing url within the node process. i.e the react client `env` file or env variable.  So if this results in `""` then in the `getAppsmithConfigs` function, we check `pricingUrl: ENV_CONFIG.pricingUrl || APPSMITH_FEATURE_CONFIGS.pricingUrl` where ENV_CONFIG.pricingUrl is the result of the above , so if its `""` the fallback is right from the window object `APPSMITH_FEATURE_CONFIGS` which has a fallback value of PROD urls.

**Review by dipyamanbiswas07 — APPROVED** [review]

**Pull request merged**